File: app/src/main/java/pl/mareklangiewicz/myblockssample/MainActivity.java
package pl.mareklangiewicz.myblockssample;

import android.os.Bundle;

import pl.mareklangiewicz.myactivities.MyActivity;

public class MainActivity extends MyActivity {

    @Override protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getGlobalNavigation().inflateHeader(R.layout.global_header);
        getGlobalNavigation().inflateMenu(R.menu.global_menu);
        if(savedInstanceState == null) {
            selectGlobalItem(R.id.fragment1);
        }
    }

}

File: app/src/main/java/pl/mareklangiewicz/myblockssample/MainActivity.kt
package pl.mareklangiewicz.myblockssample

import android.os.Bundle

import pl.mareklangiewicz.myactivities.MyActivity

class MainActivity : MyActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        gnav!!.headerId = R.layout.global_header
        gnav!!.menuId = R.menu.global_menu
        if (savedInstanceState == null) {
            gnav!!.setCheckedItem(R.id.fragment1, true)
        }
    }

}
